rocky/tools/models.py

```diff
@@ -101,6 +101,7 @@ class Meta:
             ("can_switch_organization", "Can switch organization"),
             ("can_scan_organization", "Can scan organization"),
             ("can_enable_disable_boefje", "Can enable or disable boefje"),
+            ("can_add_boefje", "Can add new or duplicated boefje"),
```

Comment:
I think this is more correct? I'll leave it up to you
```suggestion
            ("can_add_boefje", "Can add new or duplicate boefjes"),
```